drizzlepac/haputils/generate_custom_svm_param_file.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+
+""" generate_custom_svm_param_file - a module to create a template SVM processing pipeline parameter file
+    based on the observations present in the current working directory for the user to customize
+
+    Command-line USAGE: python generate_custom_svm_param_file.py [-clo] poller_filename
+
+    Optional inputs:
+    -c: If turned on, existing files with the same name as the output custom SVM parameter file created by
+    this script will be overwritten.
+
+    -l: The desired level of verboseness in the log statements displayed on the screen and written to the
+    .log file. Valid input options are 'critical', 'error', 'warning', 'info', or 'debug'.
+
+    -o: Name of the output configuration JSON file which will be created for specialized processing. This
+    file will contain ALL the input parameters necessary for processing. If not explicitly specified, the
+    default name for the output file is "custom_svm_params.json".
+
+    Python USAGE:
+    python
+    from drizzlepac.haputils import generate_custom_svm_param_file
+    generate_custom_svm_param_file.make_svm_input_file(input_filename,
+                                                       output_custom_pars_file='custom_svm_params.json',
+                                                       clobber=False,
+                                                       log_level=logutil.logging.INFO)
+
+    Important Note:
+    - This script generates only generates a template input parameter file populated with default values
```

Comment:
A couple of 'generates' in this sentence...

Reply:
Issue resolved by commit [ea3a0c1](https://github.com/spacetelescope/drizzlepac/pull/896/commits/ea3a0c1ce4eb4684be6f0c81fb0348d8e3f7b3f9)